docs/containers.conf.5.md

```diff
@@ -753,17 +753,17 @@ Indicates whether the application should be running in remote mode. This flag mo
 
 Default OCI specific runtime in runtimes that will be used by default. Must
 refer to a member of the runtimes table. Default runtime will be searched for
-on the system using the priority: "crun", "runc", "kata".
+on the system using the priority: "crun", "crun-vm", "runc", "kata".
```

Comment:
I don't think crun-vm should ever be used as a default runtime, as it requires very particular containers and doesn't run them the way other runtimes would.

Reply:
Shouldn't be an issue as people should be able to edit defaults in containers.conf. And I imagine unless crun-vm is installed, this issue also won't occur to begin with. @rhatdan wdyt?

Reply:
runc should have priority over crun-vm, but I agree that we should not be defaulting to crun-vm.

Reply:
in that case, we need to close this one, make these changes on main and then cherrypick to v0.57.

And if I'm reading default.go file correctly, crun-wasm also currently has priority over runc. Should that be changed as well while we're at it?

Reply:
PTAL https://github.com/containers/common/pull/1941 . If that looks ok, I'll add the commit here too.